app/org/sagebionetworks/bridge/models/activities/ActivityEventImpl.java

```diff
@@ -0,0 +1,40 @@
+package org.sagebionetworks.bridge.models.activities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Created by liujoshua on 6/19/2017.
+ */
+public class ActivityEventImpl implements ActivityEvent {
+    private final String healthCode;
+    private final String eventId;
+    private final String answerValue;
+    private final Long timestamp;
+
+    @JsonCreator
+    public ActivityEventImpl(@JsonProperty("healthCode") String healthCode, @JsonProperty("eventId") String eventId,
```

Comment:
health code should never be retrieved from JSON. So I would never expect to see JSON-related annotations for health code.